integration/tests/cook/util.py

```diff
@@ -177,16 +177,31 @@ def _generate_kerberos_ticket_for_user(self, username):
                       .replace('{{COOK_SCHEDULER_URL}}', retrieve_cook_url()))
         return subprocess.check_output(subcommand, shell=True).rstrip()
 
+    def _reset_auth_header(self, stop):
+        global session
+        if not stop.is_set():
+            logger.info(f'Refreshing kerberos tickets for {self.name}')
+            session.headers['Authorization'] = self._generate_kerberos_ticket_for_user(self.name)
+            threading.Timer(60.0, lambda: self._reset_auth_header(stop)).start()
+        else:
+            logger.info(f'Stopping kerberos ticket refresh for {self.name}')
+
     def __enter__(self):
         global session
         super().__enter__()
         assert self.previous_token is None
+        assert self.stop_event is None
         self.previous_token = session.headers.get('Authorization')
-        session.headers['Authorization'] = self.auth_token
+        session.headers['Authorization'] = self._generate_kerberos_ticket_for_user(self.name)
```

Comment:
Can't we remove this call since it's happening in `_reset_auth_header`?

Reply:
Done